lib/github_changelog_generator/generator/section.rb

```diff
@@ -97,14 +98,18 @@ def body_till_first_break(body)
     def issue_line_with_user(line, issue)
       return line if !@options[:author] || issue["pull_request"].nil?
 
-      user = issue["user"]
+      user = user(issue["user"])
       return "#{line} ({Null user})" unless user
```

Comment:
`GitHubChangelogGenerator::Section#user` seems to only return `String` as far as I see, so I think this line is no longer needed.

```suggestion
```